### ISSUE TITLE
Deployment parameters with 2019-04-01 schema not detected

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule.Rules.Azure#495]
**Description of the issue**

Currently only the `2015-01-01` schema version is detected as a valid parameter file schema.

i.e.

```text
https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#
```

**Expected behaviour**

Add support for `https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#` instead.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **0.16.0-B2009011**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.19041
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
